### ISSUE TITLE
Webdev server is not working. This is required for Marissa to test neuroglancer features and for development

### ISSUE THREAD

**Issue by bruce-wayne99** [ActiveBrainAtlas2/activebrainatlasadmin#148]

**Comment by bruce-wayne99** [comment]
<img width="631" alt="image" src="https://user-images.githubusercontent.com/25129274/205188730-271f9541-874f-40ad-ab94-d08935e20905.png">

**Comment by bruce-wayne99** [comment]
https://webdev.dk.ucsd.edu/activebrainatlas/neuroglancer/393

**Comment by eddyod** [comment]
I fixed the immediate problem, but webdev needs to be updated. It is using python 3.6 which is too old. The production database should be copied onto the site and these updates implemented:
https://github.com/ActiveBrainAtlas2/activebrainatlasadmin/blob/master/sql/2022-12-01.updates.sql

**Comment by drinehart1** [comment]
I did schema update (using above script) on dev-db.dk.ucsd.edu.  I then did db data synchronization from db.dk.ucsd.edu, active_atlas_production database to dev-db.dk.ucsd.edu, active_atlas_development database. 

I would recommend we consider removing active_atlas_development database from db.dk.ucsd.edu to avoid confusion.  It may also help to rename database but perhaps that is a discussion for another day.
